lib/console_yasttest.pm

```diff
@@ -30,4 +30,8 @@ sub post_run_hook {
     $self->clear_and_verify_console;
 }
 
+sub test_flags {
+    return {fatal => 0};
```

Comment:
We don't have to set it to zero, we can return empty hash.

Reply:
I think it is better to set it to 0 explicitly for a better reading/understanding, no?
But if needed, I can update the PR.

Reply:
I'm not that principled about that ;)